content/en/profiler/profiler_troubleshooting/ddprof.md

```diff
@@ -43,10 +42,14 @@ There are two capabilities you can use to override the value of `perf_event_para
 - `CAP_PERFMON`: adds BPF and `perf_event_open` capabilities (available on Linux v5.8 or later)
 
 There are a few less common permissions issues:
-- The profiler is not always able to instrument processes that change their UID on startup. This is common for many webservers and databases.
 - The profiler relies upon the `perf_event_open()` syscall, which is disallowed by some container runtimes. Check the appropriate documentation to see whether this might be the case.
 - Some seccomp profiles forbid `perf_event_open()`. If your system runs such a configuration, you may not be able to run the profiler.
 
+### "\<ERROR\> Could not mmap memory for watcher
+
+The profiler uses pinned memory to store (performance) events. This type of memory is constrained by kernel settings. You can view your current setting using `ulimit -l`. The following capability can be used to bypass this limitation:
```

Comment:
Suggestion to change the qualification.  Rather than describing how the product operates, describe the rationale (it's necessary).
```suggestion
The profiler requires pinned memory to operate. This type of memory is constrained by kernel settings. You can view your current setting using `ulimit -l`. The following capability can be used to bypass this limitation:
```